### PR TITLE
New sample with a ViewModel and multiple stores

### PR DESCRIPTION
**Pull request by fabioCollini**
This PR adds a new activity in the sample that uses a `ViewModel` that manages two stores.
The `ViewModel` exposes a `LiveData<Lce<List<Post>>>`, the activity observes it and updates the UI accordingly.  The `ViewModel` uses two stores: one to retrieve a config and the other to retrieve the real data.

### CODE REVIEW

File: app/src/main/java/com/dropbox/android/sample/Graph.kt [hunk 1]
```diff
@@ -48,10 +67,40 @@ object Graph {
...
+        val fileSystemPersister = FileSystemPersister.create(fileSystem, object : PathResolver<Unit> {
+            override fun resolve(key: Unit) = "config.json"
+        })
+        val adapter = moshi.adapter<RedditConfig>(RedditConfig::class.java)
+        return StoreBuilder
+                .fromNonFlow<Unit, RedditConfig> {
+                    delay(500)
+                    RedditConfig(10)
+                }
+                .nonFlowingPersister(
```
Comment (digitalbuddha) [inline app/src/main/java/com/dropbox/android/sample/Graph.kt]: @pavlospt this is a good starting point for what integration with `FileSystem` looks like. Feels like we can make this nicer/in a wrapper

**Comment by digitalbuddha** [comment]
Thank you!

File: app/src/main/java/com/dropbox/android/sample/Graph.kt (hunk 1 above)
Comment (pavlospt) [reply app/src/main/java/com/dropbox/android/sample/Graph.kt]: @digitalbuddha indeed. Will try to study it a bit more and give it a try with a wrapper based on this. Sorry for the delay, it's just that I have not yet been that familiar with the new Store 😛

**Comment by digitalbuddha** [comment]
No worries! We don't really have a hard date to come out of alpha. It's ready when it is ready

**Comment by digitalbuddha** [comment]
you have some ktlint violations otherwise lgtm (as long as cla is signed)

**Pull request merged**